### ISSUE TITLE
Separate Build and Test steps in CI procedure

### ISSUE THREAD

**Issue by wavefunction91** [NWChemEx-Project/DeveloperTools#82]
Build and Tests should be separated into separate steps in the CI procedure to make it easier to tell if one failed vs the other (e.g. Build can pass and Test can fail, and the only way to tell which failed is if we dig through 1000's of lines of build output to get to the testing output).

**Comment by twindus** [comment]
@wavefunction91 I don't think this has been taken care of yet.  Is that the case?

**Comment by wavefunction91** [comment]
No, not as far as I know, but it's relatively low priority, not in critical path for release.
